awx_collection/README.md

```diff
@@ -32,33 +32,11 @@ Installing the `tar.gz` involves no special instructions.
 ## Running
 
 Non-deprecated modules in this collection have no Python requirements, but
-may require the official [AWX CLI](https://pypi.org/project/awxkit/)
+may require the AWX CLI
 in the future. The `DOCUMENTATION` for each module will report this.
 
-You can specify authentication by a combination of either:
+You can specify authentication by host, username, and password.
 
- - host, username, password
- - host, OAuth2 token
-
-The OAuth2 token is the preferred method. You can obtain a token via the
-``login`` command with the AWX CLI.
-
-These can be specified via (from highest to lowest precedence):
-
- - direct module parameters
- - environment variables (most useful when running against localhost)
- - a config file path specified by the `tower_config_file` parameter
- - a config file at `~/.tower_cli.cfg`
- - a config file at `/etc/tower/tower_cli.cfg`
-
-Config file syntax looks like this:
-
-```
-[general]
-host = https://localhost:8043
-verify_ssl = true
-oauth_token = LEdCpKVKc4znzffcpQL5vLG8oyeku6
-```
```

Comment:
@tvo318 You made this change to `awx_collection/tools/roles/template_galaxy/templates/README.md.j2` but this file is templated from that file.

We have this duplication due to technical reasons, allowing installing from source. This is obtained from running `make awx_collection_build`, so I want to be super clear that this unrelated change comes from the templating.